libs/application/templates/financial-aid/src/lib/FinancialAidTemplate.ts

```diff
@@ -68,8 +68,10 @@ const FinancialAidTemplate: ApplicationTemplate<
         on: {
           SUBMIT: [
             //TODO check if works when national registry works
-            { target: ApplicationStates.DRAFT, cond: isMuncipalityRegistered },
-            { target: ApplicationStates.MUNCIPALITYNOTREGISTERED },
+            {
+              target: ApplicationStates.MUNCIPALITYNOTREGISTERED,
+              cond: isMuncipalityRegistered,
```

Comment:
Shouldn´t the condition function be named isMuncipalityNotRegistered or something like that?
isMunicipalityUnregistered?